src/components/UserInfo.js

```diff
@@ -1,7 +1,8 @@
 export default class UserInfo {
-  constructor({ nameSelector, jobSelector }) {
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._jobElement = document.querySelector(jobSelector);
+    this._avatarSelector = document.querySelector(avatarSelector);
```

Comment:
`    this._avatarSelector`


Word `selector` means a **string** that is used for searching elements in the DOM